src/Common/Core/Twig/Extensions/TwigFilters.php

```diff
@@ -5,83 +5,84 @@
 /**
  * Contains all Forkcms filters for Twig
  */
-use Twig_Environment;
-use Twig_SimpleFilter;
-use Twig_SimpleFunction;
+
+use Twig\Environment;
+use \Twig\TwigFilter;
+use \Twig\TwigFunction;
 
 class TwigFilters
 {
     /**
      * //http://twig.sensiolabs.org/doc/advanced.html#id2
      * returns a collection of Twig SimpleFilters
      *
-     * @param Twig_Environment $twig
+     * @param Environment $twig
      * @param string $app
      */
-    public static function addFilters(Twig_Environment $twig, string $app): void
```

Comment:
is this change backwards compatible?

Reply:
It's definitely not backward compatible

But I've checked the usage of TwigFilters, that only happens in the Frontend/Backend Engine
And both are rewritten to pass the new Twig\Environment class

Do extensions call the Frontend/Backend Engine them self?

Reply:
Btw, I forgot to mention that I've done some random filter tests after the Twig changes and it worked for me so far